language_formatters_pre_commit_hooks/pretty_format_yaml.py

```diff
@@ -14,6 +15,28 @@
 from six import text_type
 
 
+def process_single_document(document, yaml):
```

Comment:
I would make the function private (I don't expect that anyone uses this library as dependency of their code, but its easy to address it at this stage)

```suggestion
def _process_single_document(document, yaml):
```